apps/src/applab/applab.js

```diff
@@ -782,7 +776,8 @@ Applab.init = function (config) {
     showDebugButtons: showDebugButtons,
     showDebugConsole: showDebugConsole,
     showDebugSlider: showDebugConsole,
-    showDebugWatch: config.level.showDebugWatch || experiments.isEnabled('showWatchers')
+    showDebugWatch: config.level.showDebugWatch || experiments.isEnabled('showWatchers'),
+    debuggerUi,
```

Comment:
I'm fine with that being in a separate PR, but then might recommend a short comment explaining that this doesn't really fit the mold for pageConstants, and we intend to move it in the nearish future.